helm-charts/basehub/values.yaml

```
@@ -400,7 +404,77 @@ jupyterhub:
                   - value: "/rstudio"
                     title: RStudio
                     description: An IDE For R, created by the RStudio company
-
+    initContainers:
+      - name: templates-clone
+        image: alpine/git
+        args:
+          - clone
+          - --
+          - $(HUB_TEMPLATES_REPO)
```

```suggestion
          - ${HUB_TEMPLATES_REPO}
```

I'm not confident about this situation, thinking maybe env var expansion is trouble. It depends on if the container has `ENTRYPOINT` defined directly to `git` or `sh -c git` or similar maybe.

---

Ah nevermind, this is correct - see https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/#using-environment-variables-inside-of-your-config.